paxi/chain.json

```diff
@@ -0,0 +1,110 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "paxi",
+  "status": "live",
+  "website": "https://paxinet.io",
+  "network_type": "mainnet",
+  "chain_type": "cosmos",
+  "pretty_name": "Paxi",
+  "chain_id": "paxi-mainnet",
+  "bech32_prefix": "paxi",
+  "daemon_name": "paxid",
+  "node_home": "$HOME/.paxi",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "upaxi",
+        "fixed_min_gas_price": 0.05,
+        "low_gas_price": 0.05,
+        "average_gas_price": 0.1,
+        "high_gas_price": 0.25
+      }
+    ]
+  },
+
+  "staking": {
+    "staking_tokens": [
+      { "denom": "upaxi" }
+    ],
+    "unbonding_time": "604800s"
+  },
+
+  "codebase": {
+    "git_repo": "https://github.com/paxi-web3/paxi",
+    "genesis": {
+      "name": "paxi-mainnet",
+      "genesis_url": "https://mainnet-rpc.paxinet.io/genesis"
+    }
+  },
+
+  "images": [
```

Comment:
Please either add paxi_black.png and paxi_white.png as images somewhere or else please remove them.